dashboard/src/app/(authenticated)/(diagram)/components/NodeTypes/Sleep/SleepTaskDetails.tsx

```diff
@@ -0,0 +1,22 @@
+import { NodeRun, SleepNode } from 'littlehorse-client/proto'
+import { FC } from 'react'
+import { NodeDetails } from '../NodeDetails'
+import { convertSecondsToReadableTime } from '../../../../../utils'
+
+export const SleepTaskDetails: FC<{ sleepNode?: SleepNode; nodeRun?: NodeRun }> = ({ sleepNode, nodeRun }) => {
+  if (!sleepNode?.rawSeconds?.literalValue?.int) return
+
+  return (
+    <NodeDetails>
+      <div className="mb-2">
+        <div className="flex items-center gap-1 whitespace-nowrap text-nowrap">
+          <h3 className="font-bold">SleepTask</h3>
+        </div>
+        <div className="flex flex-col gap-2 text-nowrap">
+          <div className="flex ">Time: {convertSecondsToReadableTime(sleepNode.rawSeconds.literalValue.int)}</div>
```

Comment:
`rawSeconds` covers just one case, it ca be also:

- `timestamp`
- `iso_date`

Please verify the proto schema: https://github.com/littlehorse-enterprises/littlehorse/blob/9419e19187a8ea435dba99a776d966306072962e/schemas/littlehorse/wf_spec.proto#L433